### ISSUE TITLE
SteamUser.LoginKeyCallback not returning

### ISSUE THREAD

**Issue by cammW** [SteamRE/SteamKit#1068]
I have managed to login the user but cannot find a way to retrieve the steamLogin cookie of that session once I login.

The steamLogin cookie I need looks like [steam ID x64]%7C%7C[Token], and the token is the part I cannot find. I tried using `SteamUser.LoginKeyCallback` and get the `LoginKey` from that callback but it just ends up waiting forever for the callback to happen.

Clearly I'm doing something wrong, and I need this steamLogin cookie in order to make a post request and edit the profile I logged into.  I'm not sure whether the `LoginKey` is what I'm looking for and when I went digging into the `SteamBot` files I found the way that it's done there:
```
authResult = userAuth.AuthenticateUser(
    steamid: client.SteamID.ConvertToUInt64(),
    sessionkey: HttpUtility.UrlEncode(cryptedSessionKey),
    encrypted_loginkey: HttpUtility.UrlEncode(cryptedLoginKey),
    method: "POST",
    secure: true
);
```
```
Token = authResult["token"].AsString();
TokenSecure = authResult["tokensecure"].AsString();
```

I was not able to figure how what `AuthenticateUser` returns. If any of you is able to set me in the right direction I would greatly appreciate that

**Comment by cammW** [comment]
Quick Update: I found out what `userAuth.AuthenticateUser` returns and it's actually from the Steam Web API (https://partner.steamgames.com/doc/webapi/ISteamUserAuth). I managed to create and encrypt a session key, i have the steam id, and all that remains is the login key, which I assume is the one that `SteamUser.LoginKeyCallback` has.

The only problem now is, I still can't get that callback to work. The description says "This callback is returned some time after logging onto the network." but no matter how long I wait nothing happens
